db/version_edit.h

```diff
@@ -423,6 +423,19 @@ class VersionEdit {
   std::string DebugString(bool hex_key = false) const;
   std::string DebugJSON(int edit_num, bool hex_key = false) const;
 
+  void SetStateUponManifestSwitchTag(bool tag) {
+    state_upon_manifest_switch_ = tag;
+  }
+  bool GetStateUponManifestSwitchTag() const {
+    return state_upon_manifest_switch_;
+  }
+  void SetManifestSwitchFinishedTag(bool tag) {
+    manifest_switch_finished_ = tag;
+  }
+  bool GetManifestSwitchFinishedTag() const {
+    return manifest_switch_finished_;
+  }
+
```

Comment:
For this group of functions, I suggest that we rename them by removing `SwitchTag` or `FinishedTag`.

Reply:
@riversand963 I will rename to SetStateUponManifest/GetStateUponManifest and SetManifestSwitched/GetManifestSwitched